bigbluebutton-html5/imports/ui/components/app/AppContainer.jsx

```diff
@@ -1,11 +1,9 @@
-import React, { Component } from 'react';
+import React from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import { withModalMounter } from '../modal/service';
 
 import AppView from './AppView';
 
-export default class AppContainer extends Component {
-  render() {
-    return (
-      <AppView />
-    );
-  }
-}
+const AppContainer = ({ props }) => <AppView {...props} />;
```

Comment:
should be like

`const AppContainer = (props) => <AppView {...props} />;`

Reply:
done